server/src/test/kotlin/io/spine/server/entity/ExtensionsTest.kt

```diff
@@ -33,8 +34,14 @@ internal class ExtensionsTest {
         val entity = createEntity()
         val str = randomString()
         entity.apply(str)
-        assertThat(entity.value())
-            .isEqualTo(str)
+        assertThat(entity.value()).isEqualTo(str)
+    }
+
+    @Test
+    fun `add 'under' function for 'ServerEnvironment' configuration`() {
```

Comment:
So we're not back-ticking anymore, right?

Reply:
We cannot backtick here because the whole method name is backticked. So here I use single quotes to still highlight the code-related names, which I believe is important for readability of the programming-related texts.